src/main/java/ca/on/oicr/pineryreports/reports/impl/GazpachoProjectStatusReport.java

```diff
@@ -392,6 +400,32 @@ protected void collectData(PineryClient pinery) throws HttpResponseException, IO
     // reverse the "all" lists, as we want to display most recent items at the top
     Collections.reverse(dnaAllList);
     Collections.reverse(rnaAllList);
+
+    dnaStagnantList = getStagnant(dnaAllList);
+    rnaStagnantList = getStagnant(rnaAllList);
+  }
+
+  /**
+   * Get all the items created in the past six months which don't have recent activity (within the past month)
+   * 
+   * @param allReportItems list of all report items
+   * @return list of report items created in the past six months with no recent activity
+   */
+  private List<Entry<String, ReportItem>> getStagnant(List<Entry<String, ReportItem>> allReportItems) {
+    return allReportItems.stream().
+        filter(entry -> {
+          boolean stagnant = false;
+          try {
+            String lanesCompleted = entry.getValue().getNumLanes("Completed", null);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Completed" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)